src/global.ts

```diff
@@ -158,6 +158,26 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
    */
   useRefreshTokens?: boolean;
 
+  /**
+   * If true, fallback to the legacy technique of using a hidden iframe and the `authorization_code` grant with `prompt=none` when unable to use refresh tokens.
+   * The default setting is `true`.
+   *
+   * **Note**: There might be situations where using the legacy iframe technique is not supported, and using it as a fallback might be undesired.
```

Comment:
"There might be situations where doing silent auth with a Web Message response from an iframe is not possible, like when you're serving your application from the file system or a custom protocol (like in a Desktop or Native app). In situations like this you can disable the iframe fallback and handle the failed Refresh Grant and prompt the user to login interactively with `loginWithRedirect` or `loginWithPopup`"